mlrun/projects/project.py

```diff
@@ -1797,28 +1797,126 @@ def set_model_monitoring_application(
         **application_kwargs,
     ) -> mlrun.runtimes.BaseRuntime:
         """
-        update or add a monitoring application to the project & deploy it.
+        update or add a monitoring function to the project.
```

Comment:
Since you have taken initiative to use capital letters
```suggestion
        Update or add a monitoring function to the project.
```